docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,30 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.8.5 - November 11, 2024
+
+### Enhancements
+
+- The guardrails checks to validate source/target database permissions, verify binary dependencies, and check database version compatibility for PostgreSQL in all voyager commands is now enabled by default.
+- UI/UX improvements in the PostgreSQL permission grant script and new checks are added for replication slots, foreign keys, and triggers in PostgreSQL guardrails.
+- Object names are scrollable in the analyze schema HTML report for improved navigation.
+- Added constraint names and their corresponding table names when reporting unsupported features related to deferrable and exclusion constraints.
+- Added reporting for the REFERENCING clause for triggers and BEFORE ROW triggers on partitioned tables in the analyze-schema and assess-migration reports.
+- Added functionality to include documentation links for unsupported query constructs in the assessment report.
+- Standardized the format of data sent to the yugabyted control plane via the assess-migration command, ensuring consistent presentation across various sections of the report, such as Unsupported Features, Unsupported Datatypes, and Unsupported Query Constructs.
+
+### Bug fixes
+
+- Fixed the import-schema DDL parsing issue for functions and procedures, where extra spaces before the DDL caused it to be treated as normal DDL, preventing the PLPGSQL parsing logic from triggering.
+- Fixed an issue which resulted in "token too long" errors in export-data-from-target when log level was set to DEBUG.
+
+### Known issues
+
+- The [assess-migration](../reference/assess-migration/) command will fail if the [pg_stat_statements](../../explore/ysql-language-features/pg-extensions/extension-pgstatstatements/) extension is created in a non-public schema, due to the Unsupported Query Constructs feature.
```

Comment:
Unsupported Query Constructs are a feature? What is this actually referring to

Reply:
@aishwarya24 if we can put this `Unsupported Query Constructs` in quotes then it should be better. 
@ddhodge its a feature `assess-migration` command provides.